src/org/traccar/model/Position.java

```diff
@@ -22,9 +22,11 @@
     public static final String KEY_ORIGINAL = "raw";
     public static final String KEY_INDEX = "index";
     public static final String KEY_HDOP = "hdop";
-    public static final String KEY_SATELLITES = "sat";
+    public static final String KEY_SATELLITES = "sat"; // in use
+    public static final String KEY_SATELLITES_VISIBLE = "satVisible";
```

Comment:
I don't think we need to save this value. I doubt anybody cares about it. Also, if attribute is used only one, don't put it into `Position` class.